src/bt_peer_connection.cpp

```diff
@@ -664,7 +660,7 @@ namespace {
 		aux::write_uint32(crypto_field, write_buf);
 		aux::write_uint16(pad_size, write_buf); // len (pad)
 
-		std::generate(write_buf.data(), write_buf.data() + pad_size, random_byte);
+		aux::random_bytes({write_buf.data(), pad_size});
```

Comment:
Hi @arvidn, do you mind taking a look at the two lines before this one? I see a couple of `aux::write_uint16`, but then, this overwrite whatever is there, is that right? or am I missing something?